spread/project.go

```diff
@@ -529,7 +537,7 @@ func Load(path string) (*Project, error) {
 			backend.Type = bname
 		}
 		switch backend.Type {
-		case "google", "linode", "lxd", "qemu", "adhoc", "humbox":
+		case "google", "linode", "lxd", "qemu", "adhoc", "humbox", "openstack":
```

Comment:
Just for aesthetics, can we please move it after google?  It's the best analogous in that list.

Reply:
done